docs/UserGuide.md

```diff
@@ -241,27 +256,34 @@ Successful Output:
 
 Deletes the client contact from the contact book by their index.
 
-Format: `delete ENTRY_INDEX`
+Format: `delete INDEX`
 
 * Deletes the person at the specified `INDEX`.
 * The index refers to the index number shown in the displayed person list.
 
 Acceptable Values:
-1. ENTRY_INDEX - Number (1 to current size of the contact book).
+1. INDEX - Number (1 to current size of the contact book).
 
 Examples:
 * `list` followed by `delete 1` deletes the 1st person in the contact book.
 
 Successful Output:
-`Contact Deleted!
-Low Jun Yu is removed.`
+`Deleted Person: David; 
+Phone: 93234567; 
+Email: davidmiller@gmail.com; 
+Address: Bishan Blk 999 #08-15 569874; 
+Next-of-kin Name: Olivia; 
+Next-of-kin Phone: 56981234; 
+Appointment: Meeting, 15-12-2023 17:30; 
+Financial Plans: [Financial Plan A][Financial Plan B]; 
+Tags: `
 
 ----------
 ### Scheduling an Appointment : `schedule`
 
-Schedules an appointment for a client using an entry index followed by the appointment details. 
+Schedules an appointment for a client using an index followed by the appointment details. 
 
-Format: `schedule ENTRY_INDEX [ap/APPOINTMENT_NAME] [d/APPOINTMENT_DATE]`
+Format: `schedule INDEX ap/APPOINTMENT_NAME d/APPOINTMENT_DATE`
```

Comment:
To address the issue of inconsistent date formatting for complete and schedule comamnd, can also update **APPOINTMENT_DATE** to **APPOINTMENT_DATE_TIME**.

Reply:
This can be done in a separate PR